test/cppsuite/CMakeLists.txt

```diff
@@ -59,6 +59,8 @@ target_compile_options(
     PRIVATE ${COMPILER_DIAGNOSTIC_CXX_FLAGS}
 )
 target_link_libraries(cppsuite_test_harness PRIVATE test_util)
+# There's no special sub-path to access a given extension.
+target_compile_options(cppsuite_test_harness PUBLIC -DEXTSUBPATH=\"\")
```

Comment:
Do we need to define this variable?

Reply:
Yep, we need to define `EXTSUBPATH` to an empty string otherwise the cppsuite compilation will default the value to `.libs` (the autoconf convention).